src/test/java/part1/exercise/StreamsExercise2.java

```diff
@@ -121,4 +156,78 @@ public void greatestExperiencePerEmployer() {
         );
     }
 
+    private static Stream<StreamsExercise2.PersonEmployerPair> employeeToPairs(Employee employee) {
+        return employee.getJobHistory().stream()
+                .map(JobHistoryEntry::getEmployer)
+                .map(e -> new StreamsExercise2.PersonEmployerPair(employee.getPerson(), e));
+    }
+
+    private static Stream<StreamsExercise2.PersonEmployerPair> firstEmployeeToPairs(Employee employee) {
+        return employee.getJobHistory().stream()
+                .map(JobHistoryEntry::getEmployer)
+                .limit(1)
+                .map(e -> new StreamsExercise2.PersonEmployerPair(employee.getPerson(), e));
+    }
+
+    private Map<String, Person> getGreatestExperiencePerEmployer(List<Employee> employees) {

```

Comment:
You should prefer Person with 10 single year records over Person with a single 5 years record.